scripts/deploy/check-monitors.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Check monitors status
+ * Usage:
+ * node check-monitors.js us1,eu1,...
+ */
+const { printLog, runMain, fetch } = require('../lib/execution-utils')
+const { getTelemetryOrgApiKey, getTelemetryOrgApplicationKey } = require('../lib/secrets')
+const { siteByDatacenter } = require('../lib/datadog-sites')
+
+const monitorIdsByDatacenter = {
+  us1: [72055549, 68975047, 110519972],
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is there an API to get the list of monitors for a datacenter?

Reply:
I have not checked, I guess you are wondering if we could query those monitors instead of listing it.
I have not thought about it, I stayed close to the SDP API.
It should not change that often though.
Do you see/forsee any issue?

Reply:
Not an issue, just less maintenance: It could prevent us from updating this script if we add or remove a monitor at some point.

Reply:
FMU, It would require us to have a query specific enough to select only the monitors that we want to check before deployment, which we don't have for now.
We could attach a specific tags but it would require to update all monitors and then it could be easily missed or added inadvertently on new monitors 🤔
I think it should be good enough for now and it could be revisited if we start to add new monitors and are bothered by this configuration.